lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -158,6 +158,12 @@
   &.-active {
     display: block;
   }
+
+  @include media($mobile) {
+    .gallery.-quartet > li:nth-last-of-type(-n+2) {
```

Comment:
It'd be nice to put a little `// Hides the last two items on mobile.` comment here since the `nth-last-of-type` syntax isn't super readable at first glance.